docker/build/run

```diff
@@ -39,6 +39,10 @@ try:
 except subprocess.CalledProcessError:
     pass
 
+if 'DOCKER_HOST_' in os.environ:
+    with open('/etc/hosts', 'a') as f:
+        f.write('\n{} docker-host\n'.format(os.environ['DOCKER_HOST_']))
```

Comment:
Why do need this ? I do not found where it is used.

Reply:
Not directly but is can be useful to have it :-)